contributors/yash-raj-bharti/index.md

```diff
@@ -0,0 +1,37 @@
+---
+contributor_name: Yash Raj Bharti
+folder_name: yash-raj-bharti
+is_featured: true
+img_alt: A person standing next to a colorful graffiti-painted Android statue, with the Google headquarters in the background. The person is wearing a lanyard on a sunny day.
+usernames:
+  github: yashrajbharti
+quote: MDN resonates with my passion to build a consistent and open web, where developers can learn and grow.
+---
+
+Hi! I'm' Yash, a UXE and a Senior Mentor of the Liquid Galaxy, where I work on GIS applications, making immersive data viz projects on Google Maps, Streetview, and Earth. Liquid Galaxy is a Google-founded open-source project and was my primary reason for falling in love with open-source. I work on the web: HTML, CSS, and JS, and I believe that honing strong basics can make even the most complex tasks easy.
+
+I still remember my first contribution to MDN. I was so proud of myself. Even though the fix was small, the cheer from Will Bamberg(a contributor) at the time meant a lot. It was for a [PWA](https://github.com/mdn/content/pull/29488), and I said: iTunes value associated with the App Store for platform members of related apps has been dropped since it's misleading and currently not supported by Safari. The list has been updated as per the W3C wiki.
```

Comment:
```suggestion
I still remember my first contribution to MDN. I was so proud of myself. Even though the fix was small, the cheer from Will Bamberg at the time meant a lot. It was for [PWA docs](https://github.com/mdn/content/pull/29488), where I corrected that the iTunes value associated with the App Store for platform members of related apps has been dropped since it's currently not supported by Safari.
```